### DISCUSSION TITLE
Paperless behind authentik: can't login

### DISCUSSION THREAD

**Discussion by basbebe** [bauerj/paperless_app#106] [Q&A]
I'm using paperless behind Traefik and use [authentik](https://goauthentik.io/) to handle authentication.

The app is located at `https://paperless.domain.tld`.

I tried to allow `^/api/.*` using authentik's "Unauthenticated Paths" but I always get an error when trying to login with the app.

Even if I allow **all** paths (`^.*`), resulting in _no_ login requests when I use `curl` or the browser, I get the same error within the app:
```
Error while connecting to server
DioError [DioErrorType.response]: Http status error 401
Source stack:
# DioMixin. fetch (package:dio/src/
dio mixin.dart:473)
DioMixin.request (package:dio/src/ dio mixin.dart:468)
#2 DiMixin.get (package:dio/src/ dio mixin.dart:55)
#3
API.checkCredentials (package:paperless_app/api.dart:253)
#4
LoginRouteState.save (package:paperless_app/routes/ login_route.dart:45)
<asynchronous suspension>
```
![Error while connecting to server](https://user-images.githubusercontent.com/7083188/200548735-e7b56d5a-3c6f-4cff-958f-29940a224d40.png)

**Comment by jhakonen** [comment]
I just setup Authentik for my Paperless service and for me adding `^/api/.*` worked fine to get the app working. Also needed `^/fetch/.*` to be able to access stored documents.

That `checkCredentials` in the source stack seems to be querying /api/documents with Authorization header set. Try using curl from bash command line if you can access the same path without 401 error:
```bash
curl --header "Authorization: Basic $(echo -n "USERNAME:PASSWORD" | base64)" https://paperless.domain.tld/api/documents/
```
(replace USERNAME and PASSWORD with Paperless's user credentials (not your Authentik credentials)).
